manifests/scheduler.pp

```diff
@@ -48,9 +54,41 @@
       tag     => 'st2::config',
     }
 
+    if ($scheduler_num > 1) {
```

Comment:
These seem like the exact same thing in every file, wonder if you could make a defined resource for this instead to avoid copy/paste?